src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+@import "../utils/vars";
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  background-color: $white-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-size: $font-size-sm;
+  font-weight: $font-weight-medium;
+
+  &__link {
+    height: 40px;
+    width: 40px;
```

Comment:
You can use mixin, or at least variable.

Reply:
> You can use mixin, or at least variable.

Thank you for remarks!